README.md

```diff
@@ -56,33 +58,25 @@ Together, these tools enable detailed live monitoring and convenient post-analys
 ### `knx-lens-web.py`
 
 * **Web Access**: Makes the full functionality of `knx-lens.py` available via a web interface.
-* **Multi-User Capable**: Multiple users can access the explorer simultaneously through the browser.
+* **Multi-User Capable**: Multiple users can access the explorer simultaneously through the browser
 * **Simple Start**: A single script starts the server and makes the application available on the local network.
 
 ## Installation
 
-The tools require Python 3.8 or higher.
+The tools require Python 3.10 or higher. Linux and Windows are tested
```

Comment:
This was the only change. 

When something is fore-pushed to main, it changes the history - thus things get messed up in PRs (those are based on forks pre-force-push).